.gitignore

```diff
@@ -0,0 +1,6 @@
+.classpath
+.project
+.settings
+target
+*.iml
+.idea
```

Comment:
Those entries in here which are expected to be at the root should start with `/`. Those entries which are directories should end with `/`. See the main accumulo repo, `core/.gitignore` for example.